### ISSUE TITLE
List pages, posts and templates using a specific widget type

### ISSUE THREAD

**Issue by danielporto** [pojome/elementor#6364]
## Prerequisites

- [x ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x ] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
I'm always annoyed to have to go throughout all pages to find out where a 3rd party widget was used, and how will the design look like if the widget is broken (due an update or conflict) or disabled.

I often prefer elementor core widgets and only use 3rd party widgets as temporary solutions or when is not possible to achieve easily with elementor standard and pro widgets. Thus, when time permits, I tend to replace the usage of 3rd parties widgets for elementor supported ones for higher compatibility and then disable the 3rd parties ones. 

Problem is that I'm never sure if I look at all pages in which the 3rd party widget was used. And disabling/removing it can break the layout without warning. Currently, this fail silently, breaking the layout in pages that are not often visited (until someone issue a ticket) and I wish to be proactive in this matter.
Also as a safety check, I look at critical pages after a plugin update. But that's hard to do as the number of pages (not template defined posts) increase with time.

**Describe the solution you'd like**
I wish to quickly have a list, accessible via Elementor menu in the dashboard, to learn  where a given widget/plugin (for instance a 3rd party gallery or responsive table) is being used (single pages, or elementor templates, blocks, etc..). 

**Describe alternatives you've considered**
Download the website locally search manually. That's not ideal but works for one-time-job to fully disable something. But in case I want to replace a widget gradually (one page at the time), it is simply too much work to do often.  

**Additional context**
Simply put: disabling 3rd party widgets breaks layout silently. 
We need a way to find out where these widgets are being used to either replace them properly or sanity check after an update.

**Comment by bainternet** [comment]
Currently, this feature is not supported, but we will look into it in the future if we see a large demand for it.

Thanks for the feedback.
